src/components/DatasetDetailPage.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { FunctionComponent } from 'react'
+import {
+  withStyles,
+  createStyles,
+  WithStyles as IWithStyles
+} from '@material-ui/core/styles'
+import { Typography, Box, Tooltip } from '@material-ui/core'
+
+import Table from '@material-ui/core/Table'
+// import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Paper from '@material-ui/core/Paper'
+
+import tagToBadge from '../config/tag-to-badge'
+import InfoIcon from '@material-ui/icons/Info'
+
+import { formatUpdatedAt } from '../helpers' 
+
+import { useParams } from 'react-router-dom'
+import _find from 'lodash/find'
+import _keys from 'lodash/keys'
+
+import { IDataset } from '../types'
+
+const styles = () => {
+  return createStyles({
+    root: {
+      padding: '42vh 6% 1%',
+      position: 'absolute',
+      zIndex: 1,
+      width: '100%'
+    },
+    tagContainer: {
+      display: 'flex',
+      margin: '12px 12px 0px 0px'
+    },
+    noData: {
+      padding: '125px 0 0 0'
+    },
+    infoIcon: {
+      float: 'right',
+      paddingRight: '60px',
+      paddingTop: '3px'
+    },
+    tableCell: {
+      paddingTop: '12px'
+    },
+    paper: {
+      overflowX: 'auto',
+      marginTop: '10px'
+    },
+    updated: {
+      marginTop: '10px'
+    }
+  })
+}
+
+type IProps = IWithStyles<typeof styles> & { datasets: IDataset[] }
+
+const DatasetDetailPage: FunctionComponent<IProps> = props => {
+  const { datasets, classes } = props
+  const {
+    root, paper, updated, tagContainer, noData, infoIcon, tableCell
+  } = classes
+  const { datasetName } = useParams()
+  const dataset = _find(datasets, d => d.name === datasetName)
+  if (!dataset) {
+    return (
+      <Box
+        mt={10}
+        display='flex'
+        justifyContent="center"
+        className={root}
+      >
+        <Typography align='center' className={noData}>
+          No dataset by the name of <strong>&quot;{datasetName}&quot;</strong> found
+        </Typography>
+      </Box>
+    )
+  } else {
+    const {
+      name,
+      description,
+      tags = [],
+      updatedAt
+    } = dataset
+
+    let {
+      fields
+    } = dataset
+    
```

Comment:
What happens if fields is empty array?

Reply:
there's no header, it's just an empty "Paper" component box